directory/datadog_checks/directory/directory.py

```diff
@@ -151,3 +165,4 @@ def _get_stats(self):
         # total file size
         if not self._config.countonly:
             self.gauge('system.disk.directory.bytes', directory_bytes, tags=dirtags)
+            self.log.debug('`countonly` not enabled': Collecting system.disk.directory.bytes metric.)
```

Comment:
Oops! My suggestion didn't have correct syntax. Fixed now:
```suggestion
            self.log.debug("`countonly` not enabled: Collecting system.disk.directory.bytes metric.")
```